worldedit-core/src/main/java/com/sk89q/worldedit/WorldEditException.java

```diff
@@ -19,11 +19,19 @@
 
 package com.sk89q.worldedit;
 
+import com.sk89q.worldedit.util.formatting.WorldEditText;
+import com.sk89q.worldedit.util.formatting.text.Component;
+import com.sk89q.worldedit.util.formatting.text.TextComponent;
+
+import java.util.Locale;
+
 /**
  * Parent for all WorldEdit exceptions.
  */
 public abstract class WorldEditException extends Exception {
 
+    private Component message;
```

Comment:
Adding a new field that is not final when it should be :(

Reply:
It's not always initialised so therefore isn't final